lib/hets-agent/application.rb

```diff
@@ -41,6 +47,41 @@ def id
       def normalize_paths
         Settings.hets.path = Pathname.new(Settings.hets.path)
       end
+
+      def initialize_bunny
+        username = Settings.rabbitmq.username
+        password = Settings.rabbitmq.password
+        host = Settings.rabbitmq.host || 'localhost'
+        port = Settings.rabbitmq.port || 5672
+        connection_string = "amqp://#{username}:#{password}@#{host}:#{port}"
+        @bunny = Bunny.new(connection_string)
+      end
+
+      def initialize_version_requirement
+        @hets_version_requirement =
+          HetsAgent::VersionRequirementRetriever.new.call
+
+        return unless hets_version_requirement.nil?
+
+        message =
```

Comment:
Does that mean we have to start the backend before all workers? I think it would be nicer to block until a message is received (if that is somehow possible).

Reply:
Fixed in e92af48